spec/feature/authenticated_user_spec.rb

```diff
@@ -168,29 +168,40 @@
     end
   end
 
-  it "can view past orders with links to each order" do
+  it "can view past trips with links to each trip" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
                                                     and_return(valid_user)
     Order.create(user_id: valid_user.id)
     Order.create(user_id: valid_user.id)
     visit user_path(valid_user.id)
-    click_link_or_button "View past orders"
+    click_link_or_button "My Trips"
     expect(current_path).to eq(orders_path)
     within(".orders-list") do
       expect(page).to have_content("Order 00001")
     end
   end
 
-  it "can view particular orders (order show page)" do
+  it "can view particular trips (trip show page)" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
     and_return(valid_user)
     Order.create(user_id: valid_user.id)
     visit user_path(valid_user.id)
-    click_link_or_button "View past orders"
+    click_link_or_button "My Trips"
     click_link_or_button "Order 00001"
     expect(page).to have_content("Order 00001")
   end
 
+  it "can edit their profile on their own page" do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+                                                    and_return(valid_user)
+    visit user_path(valid_user)
+    expect(current_path).to eq(user_path(valid_user))
+    click_link_or_button('Edit Profile')
+    fill_in "user[about_me]", with: "I'm no fun!"
+    click_link_or_button 'Submit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.